### PR TITLE
i18n: adds Occitan locale

### PR DESCRIPTION
**Pull request by Mejans**
I hope I made all the required changes

### CODE REVIEW

File: assets/i18n/oc.po [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+msgstr "Seleccionar una opcion"
+
+msgid "Select version for {0}"
+msgstr "Seleccionar una version per {0}"
+
+msgid "Select..."
+msgstr "Seleccionar..."
+
+msgid "SPICE port"
+msgstr "Pòrt SPICe"
```
Comment (philclifford) [inline assets/i18n/oc.po:100]: ?Small typo s/SPICe/SPICE/

Comment (Mejans) [reply assets/i18n/oc.po:100]: yes :s

**Review by philclifford — APPROVED** [review]
Thanks! :+1:

**Comment by philclifford** [comment]
You might want to look at #42 and consider if you can present @ymauray  with an option with and without the extra string that might be needed if/when #42 is merged ...

**Comment by Mejans** [comment]
Hello thanks for mencioning this, I cannot check the life of the repo after my PR ^^
It would be:
msgid "Download cancelled"
msgstr "Telecargament anullat"

**Comment by philclifford** [comment]
Thanks: I'll try ...

**Pull request merged**